api_client/python/timesketch_api_client/client.py

```diff
@@ -434,6 +434,12 @@ def list_sketches(self, per_page=50, scope='user', include_archived=True):
         Args:
             per_page: Number of items per page when paginating. Default is 50.
             scope: What scope to get sketches as. Default to user.
+                user: sketches owned by the user
+                recent: Get list of sketches that the user has actively searched in.
+                shared: Get sketches that can be accessed
+                admin: pre-sets some base filters
```

Comment:
Worth noting that admin only works if you're an admin, otherwise 0 sketches are returned

Reply:
```suggestion
                admin: Get all sketches if the user is an admin 
```